moto/cloudwatch/models.py

```diff
@@ -677,33 +676,6 @@ def _get_paginated(self, metrics):
             return None, metrics
 
 
-class LogGroup(CloudFormationModel):
```

Comment:
Unrelated bug that only popped up after adding the 'has_cfn_attribute`-methods.
This class would be invoked by CF, but it returns another class (moto/logs/LogGroup) that did not extend CloudFormationModel - so responsibility for the creation was split between the two.

I've reworked this so that the `moto/logs/LogGroup`-class is completely responsible for CF creation.